scripts/profile-compiler-build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# To be run from the root of the Flambda backend repo
+
+set -e -u -o pipefail
+
+dump_dir="`pwd`/profile"
+summary_path="`pwd`/summary.csv"
+
+export OLD_OCAMLPARAM="${OCAMLPARAM:-}"
+export OCAMLPARAM="_,profile=1,dump-into-csv=1,dump-dir=$dump_dir,regalloc=irc,regalloc-param=SPLIT_LIVE_RANGES:on,regalloc-param=IRC_SPILLING_HEURISTICS:flat-uses"
+export BUILD_OCAMLPARAM="$OCAMLPARAM"
+
+revert_env_variables() {
+  export OCAMLPARAM="$OLD_OCAMLPARAM"
+  unset OLD_OCAMLPARAM
+}
+trap revert_env_variables EXIT
```

Comment:
Probably not necessary.

Reply:
It seems like a good feature to have as someone might wonder why their compiler keeps producing CSVs after running this script.

Reply:
I don't think the export can be seen
after running the script.

Reply:
Ah I see the environment variables set inside the script do not persist outside.